src/handlers.ts

```diff
@@ -24,6 +32,16 @@ class Handlers {
         await this.jibriTracker.track(status);
         res.sendStatus(200);
     }
+
+    async sidecarPoll(req: Request, res: Response): Promise<void> {
```

Comment:
Using the term "poll" here seems odd as it indicates there's a loop doing some polling somewhere and I am missing that. I imagine some other polling loop exists somewhere and is calling this api/handler? If so, I suggest changing the name to indicate what it's actually doing, probably handleShutdownStatus or checkShutdownStatus?